lib/src/main/java/graphql/nadel/engine/transform/hydration/NadelHydrationTransform.kt

```diff
@@ -145,6 +145,7 @@ internal class NadelHydrationTransform(
         return makeTypeNameField(
             aliasHelper = state.aliasHelper,
             objectTypeNames = objectTypeNames,
+            deferredExecutions = linkedSetOf(),
```

Comment:
Does that mean that, for now, hydrations (batched and non-batched) won't work when nested inside deferred fields?

Reply:
Should this be 
```        
        return makeTypeNameField(
            aliasHelper = state.aliasHelper,
            objectTypeNames = objectTypeNames,
            deferredExecutions = field.deferredExecutions,
        )
        ```
instead?